CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,26 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    /// Override this property if you need to set a specific font
+    /// Defaults to 16 without an icon and 10 with
+    public var titleFont: UIFont? = nil {
+        didSet {
+            updateTabs()
+        }
+    }
+    
     @IBInspectable public var lightBackground: Bool = true {
         didSet {
             updateBackground()
         }
     }
     
+    @IBInspectable public var translucentBackground: Bool = false {
```

Comment:
But when this is `true` it cancels out the light and dark background blurs. Is that what you want the default behavior to be?